server/database/models/author.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Author = sequelize.define('Author', {
+    fullname: DataTypes.STRING
+  }, {});
+  Author.associate = function(models) {
+    // associations can be defined here
+  };
```

Comment:
Since you're creating the books model as well, the Authors model should be associated with it.

Reply:
> Since you're creating the books model as well, the Authors model should be associated with it.

It's the book model that's associated with the authors model from the digram.  The author doesn't have association with the book model

Reply:
Associations are a two-way thing. From the diagram, a book 'belongsTo' one author, while an author can have many books, so a 'hasMany' association ought to be there.

Reply:
so you are saying I should add a bookid to the author's table?

Reply:
No, the author already has an authorId in the book table.

Reply:
yeah so what do i link the new association to?
author (id) to book(authorId)?

Reply:
Yes